extension/firefox/Makefile

```
@@ -53,5 +55,24 @@ ensureoutputdir: clean
 clean:
 	-rm -rf build
 	-rm -rf $(BUILD_DIR)
+	-rm -rf $(RESTARTLESS_DIR)
 
-PHONY: build clean ensureoutputdir
+restartless:
+	-rm -rf $(RESTARTLESS_DIR)
```

first minus? no, it's just ignoring the error code